tc/sgx/trusted_worker_manager/enclave/signup_enclave.cpp

```diff
@@ -218,46 +132,6 @@ tcf_err_t ecall_CreateSignupData(const sgx_target_info_t* inTargetInfo,
     return result;
 }  // ecall_CreateSignupData
 
-// XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-tcf_err_t ecall_UnsealEnclaveData(
```

Comment:
Removing these functions from here should have caused breakage. I do not see the refactored files where these functions were being used.

Reply:
It should not break, because SGX utils will automatically route the API calls to the location where they are defined.